client/src/components/ResultsBox.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+class ResultsBox extends Component {
+    constructor(props) {
+        super(props);
+        console.log( JSON.stringify(props));
+    }
+    
+    render() {
+        const results = this.props.podcasts.map( (inst, i) => {
```

Comment:
Remove this function from the render() method. Add it to a separate method, such as a renderResults() method.